crates/breez-sdk/core/src/sdk.rs

```diff
@@ -174,7 +177,8 @@ pub async fn connect(request: crate::ConnectRequest) -> Result<BreezSdk, SdkErro
         .join(path_suffix);
 
     let storage = default_storage(storage_dir.to_string_lossy().to_string())?;
-    let builder = crate::SdkBuilder::new(request.config, request.seed, storage);
+    let mut builder = super::sdk_builder::SdkBuilder::new(request.config, request.seed, storage);
+    builder = builder.with_real_time_sync(BREEZ_SYNC_SERVICE_URL.to_string());
```

Comment:
I'm wondering if we need the option to run without the sync like in the Liquid SDK? The Breez itests run on regtest without an API key, so it would cause errors authenticating with the default sync service url. Could the sync service url be an optional config field and the sync service be initialized in the SDK instead of the builder?

Reply:
I made the url an optional field. Initializing in the SDK instead of the builder became a little more problematic now, because this function now also takes sync storage as a parameter.

Reply:
Can you check whether the current state has what you want here @dangeross? Currently on `connect`, if the url is `None`, I initialize it with the default. Should I do that in `default_config` instead?

Reply:
I think ideally we want:
- Real-time sync enabled by default (including using the basic `connect()` initialization)
- Can be opted-out by setting the config param to None
- Can be customized with a different storage implementation using the SDK builder

What do you think @roeierez?